### PR TITLE
Add requirements, readme, and docker-compose db

### PR DESCRIPTION
**Pull request by exterkamp**

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+pandas
```
Comment (zdiemer) [inline src/requirements.txt:1]: These should at least be pinned to versions for easier reproducibility, optionally could generate using [`--require-hashes`](https://pip.pypa.io/en/stable/topics/secure-installs/#hash-checking-mode).

File: app/scripts/parse_csv.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                row["cmplt_time"] = float(cmplt_time)
+
+            row["Rating"] = float(row["Rating"].replace("%", "") if row["Rating"] else 0) / 100
+            row["metacritic_rating"] = float(row["Metacritic Rating"].replace("%", "") if row["Metacritic Rating"] else 0) / 100
+            row["gamefaqs_rating"] = float(row["GameFAQs User Rating"].replace("%", "") if row["GameFAQs User Rating"] else 0) / 100
+
+            row["Priority"] = float(row["Priority"]) if row["Priority"] else 0
+            row["Wishlisted"] = "TRUE" if row["Wishlisted"] == 1 else "FALSE"
+
+            row["est_time"] = float(row["Estimated Time"]) if  not row["Estimated Time"] == " " and not row["Estimated Time"] == "" else 0
```
Comment (zdiemer) [inline app/scripts/parse_csv.py:80]: nit: additional space after `if`.

File: app/scripts/parse_csv.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                    datestr = datetime.strptime(cmplt_date, '%m/%Y').strftime('%Y-%m')
+                    row["processed_cmplt_date"] = '"%s"' % (datestr)
+
+            cmplt_time = row["Completion Time"]
+            if cmplt_time == "":
+                row["cmplt_time"] = "NULL"
+            else:
+                row["cmplt_time"] = float(cmplt_time)
+
+            row["Rating"] = float(row["Rating"].replace("%", "") if row["Rating"] else 0) / 100
```
Comment (zdiemer) [inline app/scripts/parse_csv.py:73]: Rating and priority columns should be unset for missing values, not zero.

**Review by zdiemer — CHANGES_REQUESTED** [review]
Some blocking changes on the DB structure and parsing, but otherwise LGTM.

File: app/scripts/parse_csv.py (hunk 2 above)
Comment (zdiemer) [inline app/scripts/parse_csv.py:80]: Should also null if not set.

File: app/db/01_schema.sql [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+    rating                  DECIMAL,
+    metacritic_rating       DECIMAL,
+    gamefaqs_user_rating    DECIMAL,
+    notes                   VARCHAR(256)                      DEFAULT '',
+    priority                INT                            DEFAULT 0,
+    wishlisted              BOOLEAN                      DEFAULT false,
+    estimated_time DECIMAL,
+    playing_status INT                      DEFAULT 0,
+    release_year YEAR                       NOT NULL,
+    weighted_score DECIMAL,
```
[the remark this answers is not in the document]
Comment (exterkamp) [reply app/db/01_schema.sql]: Deleted them then, we can generate them.

File: src/requirements.txt (hunk 1 above)
Comment (exterkamp) [reply src/requirements.txt]: good TODO for you

**Pull request merged**